File: Juan/P3/Java/Napakalaki/src/NapakalakiGame/Prize.java
package NapakalakiGame;

/**
 *
 * @author juane
 * @version 3.1
 * Clase Prize para el buen rollo
 */
public class Prize {
    
    /* Atributos  */
    
    private int treasures;
    private int level;

    /**
     * Constructor
     */
    
    /**
     * Constructor Prize(int t, int l)
     * @param t número de tesoros
     * @param l número de niveles
     */
    Prize(int t, int l) {
        this.treasures = t;
        this.level = l;
    }

    /**
     * Consultores
     */
    
    /**
     * Método getTreasures()
     * Devuelve el número de tesoros
     * @return treasures de tipo int
     */
    public int getTreasures() {
        return this.treasures;
    }

    /**
     * Método getLevel()
     * Devuelve el número de niveles
     * @return level de tipo int
     */
    public int getLevels() {
        return this.level;
    }

    @Override
    public String toString() {
        return "Treasures = " + Integer.toString(this.treasures) 
                + ", levels = " + Integer.toString(this.level) ;
    }
}

File: Juan/P3/Ruby/Napakalaki/pkg/Napakalaki-0.0.1/lib/Prize.rb
#encoding: utf-8
#author: Juane
#Versión 3.0
require_relative 'TreasureKind.rb'
require_relative 'Monster.rb'
require_relative 'BadConsequence.rb'
module NapakalakiGame
class Prize    
    
    def initialize(treasures,level)
        @treasures = treasures
        @levels = level
    end
 
     #Método de lectura, get
    attr_reader :treasures
    attr_reader :levels
  
  
    #Método toString
 
    def to_s
        "Treasures =  #{@treasures} Niveles ganados =   #{@levels}"
    end
  
  
    #Método solo lectura
    # attr_reader	
    #Método solo escritura
    # attr_writer
end
end